tests/nlu/base/test_server.py

```diff
@@ -213,60 +231,73 @@ def test_evaluate_invalid_project_error(app, rasa_default_train_data):
     rjs = yield response.json()
     assert response.code == 500, "The project cannot be found"
     assert "error" in rjs
-    assert rjs["error"] == "Project project123 could not be found"
+    assert rjs["error"] == "Project 'project123' could not be found"
 
 
 @pytest.inlineCallbacks
-def test_evaluate_internal_error(app, rasa_default_train_data):
-    response = app.post("http://dummy-uri/evaluate",
-                        json={"data": "dummy_data_for_triggering_an_error"})
-    time.sleep(3)
+def test_evaluate_no_model(app):
+    response = app.post("http://dummy-uri/evaluate")
+    time.sleep(2)
     app.flush()
     response = yield response
     rjs = yield response.json()
-    assert response.code == 500, "The training data format is not valid"
+    assert response.code == 500, "No model in project 'default' to evaluate"
     assert "error" in rjs
-    assert "Unknown data format for file" in rjs["error"]
 
 
 @pytest.inlineCallbacks
-def test_evaluate(app, rasa_default_train_data):
+def test_evaluate_internal_error(app, rasa_default_train_data):
     response = app.post("http://dummy-uri/evaluate",
-                        json=rasa_default_train_data)
+                        json={"data": "dummy_data_for_triggering_an_error"})
     time.sleep(3)
     app.flush()
     response = yield response
     rjs = yield response.json()
-    assert response.code == 200, "Evaluation should start"
-    assert "intent_evaluation" in rjs
-    assert "entity_evaluation" in rjs
-    assert all(prop in rjs["intent_evaluation"] for prop in ["report",
-                                                             "predictions",
-                                                             "precision",
-                                                             "f1_score",
-                                                             "accuracy"])
+    assert response.code == 500, "The training data format is not valid"
 
 
 @pytest.inlineCallbacks
 def test_unload_model_error(app):
-    project_err = "http://dummy-uri/models?project=my_project&model=my_model"
+    project_err = "http://dummy-uri/models" \
```

Comment:
Let's wrap this string in `(...)`

Reply:
Elipses?  Not `"""`?  I'm not sure what you mean here

Reply:
If you use brackets you don't have to use `\`